src/app/src/pages/eval/components/Eval.tsx

```diff
@@ -114,6 +124,17 @@ export default function Eval({ fetchId }: EvalOptions) {
     // is a global store.
     resetFilters();
 
+    // Check for filter params in the URL
+    const pluginParam = searchParams.get('plugin');
```

Comment:
Document why this exists here. I don't understand it as another developer in the project.